lib/puppet/type/ini_setting.rb

```diff
@@ -49,7 +48,7 @@ def munge_boolean_md5(value)
     desc 'The ini file Puppet will ensure contains the specified setting.'
     validate do |value|
       unless Puppet::Util.absolute_path?(value)
-        raise(Puppet::Error, "File paths must be fully qualified, not '#{value}'")
+        raise(Puppet::Error, _("File paths must be fully qualified, not '%{value}'")) % { value: value }
```

Comment:
Small error here, only noticed it due to the new test.
The second ')' should be after the final '}'.
i.e. the line should be:
`raise(Puppet::Error, _("File paths must be fully qualified, not '%{value}'") % { value: value })`